Makefile

```diff
@@ -496,10 +496,14 @@ $(RELEASE_DIR_BIN)/%:
 # Utilities
 ###############################################################################
 # TODO: stop using bin/helm as an entrypoint in build scripts.
-bin/helm: helm-deps
-	@echo 'make bin/helm' is deprecated. Please use 'make helm-deps' instead.
+bin/helm: bin/helm3
```

Comment:
No reason for this target to depend on `bin/helm3`

Reply:
While technically correct, a lot of our build infra runs `make bin/helm` as an entrypoint to "install helm dependencies". Until we update the caller scripts to use the new `make helm-deps` target, I need to piggyback off of that.